sql-cli/sql_cli/dag_generator.py

```diff
@@ -108,3 +108,34 @@ def generate_dag(directory: Path, dags_directory: Path) -> Path:
         output_file=output_file,
     )
     return output_file
+
+
+def generate_render_dag(directory: Path, dags_directory: Path) -> Path:
+    """
+    Generate a DAG from SQL files.
+
+    :params directory: The directory containing the raw sql files.
+    :params dags_directory: The directory containing the generated DAG.
+
+    :returns: the path to the DAG file.
+    """
+    if not directory.exists():
+        raise SqlFilesDirectoryNotFound("The directory does not exist!")
+    output_file = dags_directory / f"{directory.name}.py"
+    sql_files = sorted(get_sql_files(directory, target_directory=dags_directory))
+    sql_files_dag = SqlFilesDAG(
+        dag_id=directory.name,
+        start_date=datetime(2020, 1, 1),
+        sql_files=sql_files,
+    )
+    render(
+        template_file=Path("templates/render_dag.py.jinja2"),
+        context={
+            "dag": sql_files_dag,
+            "dag_id": directory.name,
+            "start_date": datetime(2020, 1, 1),
+            "project_path": directory,
+        },
```

Comment:
```suggestion
        context={
            "dag": sql_files_dag,
        },
```
Since you now still need the `sql_files_dag`, I would suggest you only pass the dag as context.

Reply:
And after removing this, the only part that is different is the `template_file`. Maybe it makes sense to pass the template_file as argument to the function OR merging the template files into one. WDYT?

Reply:
@feluelle done!